<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BGLBG1TESTPRN1" name="Default test" tests="2" failures="1" timestamp="17 Feb 2016 13:23:20 GMT" time="54.546" errors="0">
  <testcase name="reportAbug" time="14.705" classname="scripts.DriverScript"/>
  <testcase name="feature" time="14.226" classname="scripts.DriverScript">
    <failure type="java.lang.AssertionError" message="VALIDATION: Feature header is present &amp;apos; DOES NOT match with actual value &amp;apos;Features">
      <![CDATA[java.lang.AssertionError: VALIDATION: Feature header is present ' DOES NOT match with actual value 'Features
	at com.testng.SonnetTestNGAssertions.throwException(SonnetTestNGAssertions.java:259)
	at com.testng.SonnetTestNGAssertions.equals(SonnetTestNGAssertions.java:90)
	at test.webdriver.SonnetAction.performAssertion(SonnetAction.java:748)
	at test.webdriver.SonnetAction.validate(SonnetAction.java:678)
	at pageClass.pageView.navigateToFeaturetTab(pageView.java:68)
	at scripts.DriverScript.feature(DriverScript.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Default test -->
